src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -34,35 +34,68 @@ let nextId = 0;
 export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
 
   /**
-   * Hidden label for screen readers.
+   * Defines an ARIA label for the checkbox to support
+   * [accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility)
+   * when the checkbox does not include a visible label. This property must be set if
+   * you are using an icon checkbox. If the checkbox includes a visible label on the screen, use the `labelledBy` property instead.
    */
   @Input()
   public label: string;
 
   /**
-   * Id of label for the checkbox.
+   * Sets the checkbox's `aria-labelledby` attribute to support
+   * [accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility).
```

Comment:
Same note as previous accessibility link.

Reply:
Added this to the Accessibility section.